healthcheck-sofa-boot-starter/src/main/java/com/alipay/sofa/healthcheck/core/AfterReadinessCheckCallbackProcessor.java

```diff
@@ -0,0 +1,118 @@
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.Status;
+import org.springframework.context.ApplicationContext;
+import org.springframework.util.Assert;
+import org.slf4j.Logger;
+
+import java.util.*;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * Used to check {@link ReadinessCheckCallback}
+ *
+ * @author liangen
+ * @author qiong.zql
+ * @version 2.3.0
+ */
+public class AfterReadinessCheckCallbackProcessor {
+
+    private static Logger                                 logger                  = SofaBootHealthCheckLoggerFactory
+                                                                                      .getLogger(AfterReadinessCheckCallbackProcessor.class);
+
+    private ObjectMapper                                  objectMapper            = new ObjectMapper();
+
+    private AtomicBoolean                                 isInitiated             = new AtomicBoolean(
+                                                                                      false);
+    @Autowired
+    private ApplicationContext                            applicationContext;
+
+    private LinkedHashMap<String, ReadinessCheckCallback> readinessCheckCallbacks = null;
+
+    public void init() {
+        if (isInitiated.compareAndSet(false, true)) {
+            Assert.notNull(applicationContext, "Application must not be null");
+            Map<String, ReadinessCheckCallback> beansOfType = applicationContext
+                .getBeansOfType(ReadinessCheckCallback.class);
+            readinessCheckCallbacks = HealthCheckUtils.sortMapAccordingToValue(beansOfType,
+                applicationContext.getAutowireCapableBeanFactory());
+
+            StringBuilder applicationCallbackInfo = new StringBuilder();
+            applicationCallbackInfo.append("Found ").append(readinessCheckCallbacks.size())
+                .append(" ReadinessCheckCallback implementation:");
+            for (String beanId : beansOfType.keySet()) {
+                applicationCallbackInfo.append(beanId).append(",");
+            }
+            logger.info(applicationCallbackInfo.deleteCharAt(applicationCallbackInfo.length() - 1)
+                .toString());
+        }
+    }
+
+    public boolean afterReadinessCheckCallback(Map<String, Health> healthMap) {
+        logger.info("Begin ReadinessCheckCallback readiness check");
+        Assert.notNull(readinessCheckCallbacks, "ReadinessCheckCallbacks must not be null.");
+
+        boolean result = true;
+        for (Map.Entry<String, ReadinessCheckCallback> entry : readinessCheckCallbacks.entrySet()) {
+            result = doHealthCheckCallback(entry.getKey(), entry.getValue(), healthMap) && result;
+        }
+
+        if (result) {
+            logger.info("SofaBootAfterReadinessCheckCallback readiness check result: success.");
+        } else {
+            logger.error("SofaBootAfterReadinessCheckCallback readiness check result: failed.");
```

Comment:
Log contents in this method do not have coincident style, should make them clear and meaningful after the classname being refactored.